Misc/NEWS.d/next/Core and Builtins/2017-10-15-23-44-57.bpo-31786.XwdEP4.rst

```diff
@@ -0,0 +1,4 @@
+Fix an issue that affects select.poll.poll and related functions in the
+select module. Now the affected functions will block for an infinite amount
```

Comment:
I propose to rephrase the NEWS entry:

Fix timeout rounding in the select module to round correctly negative timeouts between -1.0 and 0.0. The functions now block waiting for events as expected. Previously, the call was incorrectly non-blocking. Patch by Pablo Galindo.

Reply:
Between -1e-9 and 0.0 or between -1e-6 and 0.0 for different methods.

Reply:
I don't think that a NEWS entry has to be too precise. The range depends on the function. For poll, it's up to -1 ms, no?